docs/tutorial/index.md

```diff
@@ -23,110 +33,44 @@ Output logs are included in this tutorial when instructive but are typically omi
 
 WSL can be installed directly from the [Microsoft Store](https://www.microsoft.com/store/productId/9P9TQF7MRM4R).
 
-If you already have WSL installed you are advised to backup `~\.wslconfig` and remove it before continuing the tutorial:
+If you already have WSL installed and have `~\.wslconfig` on your system you
+are advised to backup the file and remove it before continuing the tutorial:
 
 ```text
+PS C:\Users\me\tutorial> Test-Path -Path "~\.wslconfig"
```

Comment:
We don't tell the user what they should do depending on the result of this command.